packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -459,11 +490,33 @@ export class PluginLedgerConnectorBesu
         return { transactionReceipt: ledgerReceipt };
       default:
         throw new Error(
-          `${fnTag} Unrecognized ReceiptType: ${req.consistencyStrategy.receiptType}`,
+          `${fnTag} Unrecognized ReceiptType: ${request.consistencyStrategy.receiptType}`,
         );
     }
   }
 
+  public async transactPrivate(options: any): Promise<RunTransactionResponse> {
```

Comment:
@travis-payne How much effort would it be to narrow the type here from `any` to something more specific? Assuming the worst case scenario is we could add an interface in our local web3-eea typings that inherits from the web3 tx config and then adds the privateFor/From parameters (and a couple of other ones I saw in your code being used)